package.json

```
@@ -34,7 +34,7 @@
     "postcss": "8.4.21",
     "postcss-cli": "10.1.0",
     "prettier": "2.8.2",
-    "vanilla-framework": "3.15.1",
+    "vanilla-framework": "4.11.0",
```

```suggestion
    "vanilla-framework": "4.14.0",
```
We can bump this up to the [most recent version of Vanilla](https://vanillaframework.io/docs/whats-new), make sure nothing breaks elsewhere